lib/randexp.js

```diff
@@ -236,12 +236,16 @@ module.exports = class RandExp {
    */
   static randexp(regexp, m) {
     var randexp;
-    if (regexp._randexp === undefined) {
-      randexp = new RandExp(regexp, m);
-      regexp._randexp = randexp;
+    var regex = regexp instanceof RegExp 
```

Comment:
I feel like this diff would look cleaner if this line overwrite the variable `regexp`, instead of creating a new one.

Reply:
Done 🙂